### ISSUE TITLE
project.json file schema warnings

### ISSUE THREAD

**Issue by 5cover** [dotnet/vscode-dotnet-runtime#1831]
### Describe the bug.

I understand that a file named `project.json` may have some significance if the extension enforces a schema on it, but an option to disable this behavior would be nice.

I have a file named `project.json` that is not related to .NET in any way in my workspace and the extension creates false positive warnings on it.

### Please attach log file(s) here if you have any.

(no logs)

### Reproduction Steps

Create a file named project.json in the current workspace which does not respect the expected schema.

### Expected Behavior

No warning is shown

### Exceptions & Errors (if any)

JSON schema warnings on the file such as `incorrect. "object" expected`

### .NET Install Tool Version

v2.0.5

### Your operating system? Please provide more detail, such as the version and or distro above.

Linux

### VS Code Version

1.90.0

**Comment by nagilson** [comment]
Hm, I think this is coming from either C# or C# DevKit. are you using C# DevKit?

**Comment by 5cover** [comment]
> Hm, I think this is coming from either C# or C# DevKit. are you using C# DevKit?

Yes I am, but VSCode Extension Bisect pointed me to the .NET Install Tool extension.

**Comment by 5cover** [comment]
Oh ok then.
